src/Info.ts

```diff
@@ -83,7 +147,26 @@ export default class Info {
             const name = args[1];
             const text = args.splice(2).join(" ");
 
-            message.channel.send(this.addInfo(name, text));
+            message.channel.send("What category would you like to put it in?")
+                .then(async (reply) => {
```

Comment:
Why not just `await` the message.channel.send, instead of using a then callback?

Reply:
Fixed